templates/database_cleaner_rspec.rb

```diff
@@ -1,21 +1,28 @@
 RSpec.configure do |config|
+  config.use_transactional_fixtures = false
+
   config.before(:suite) do
-    DatabaseCleaner.clean_with(:deletion)
+    DatabaseCleaner.clean_with(:truncation)
   end
 
   config.before(:each) do
     DatabaseCleaner.strategy = :transaction
   end
 
-  config.before(:each, js: true) do
-    DatabaseCleaner.strategy = :deletion
+  config.before(:each, type: :feature) do
+    driver_shares_db_connection_with_specs =
+      Capybara.current_driver == :rack_test
+
+    unless driver_shares_db_connection_with_specs
+      DatabaseCleaner.strategy = :truncation
+    end
```

Comment:
Does this fail if we're using debug mode in Capybara, etc? Is there a reason we shouldn't just use `js: true` from spec metadata as our criteria for this strategy?